### DISCUSSION TITLE
Deceptive site warning on authgearapps.com

### DISCUSSION THREAD

**Discussion by jwojnowski** [authgear/authgear-server#3209] [Get Help] [answered]
Hi!

I'm trying AuthGear out using the default domain (`some-project-id.authgearapps.com`).

When I'm redirected to the sign up/sign in page from my application, I'm receiving a Deceptive Site warning from the browser:

<img width="630" alt="Screenshot 2023-07-15 at 12 01 18" src="https://github.com/authgear/authgear-server/assets/29680262/932c817a-f91f-46d2-9776-a6a5e78a5ede">

This seems consistent across WebKit browsers (Brave, Chrome, Safari) and it looks like it is set for the whole `authgearapps.com`: [report from Google](https://transparencyreport.google.com/safe-browsing/search?url=authgearapps.com&hl=en).

Is there a way of dealing with it that does _not_ include using custom domain?

**Comment by chpapa** [comment, marked as answer]
@jwojnowski Sorry for late reply, we are aware of the issue as well and have reported to google safe browsing for the problem.

Meanwhile, we have registered another domains as a temporarily measure. You can go to "Custom Domain" of the portal and activate the alternative domains (authgear-apps.com) which shall solve the problem.

**Comment by chpapa** [comment]
Just an update -- as of 20 Aug 2023, Google Safe Browsing have appeared and marked the older domains `authgearapps.com` safe again.
